src/from_substrait.cpp

```diff
@@ -492,22 +492,59 @@ shared_ptr<Relation> SubstraitToDuckDB::TransformFilterOp(const substrait::Rel &
 	return make_shared_ptr<FilterRelation>(TransformOp(sfilter.input()), TransformExpr(sfilter.condition()));
 }
 
+const google::protobuf::RepeatedField<int32_t>& GetOutputMapping(const substrait::Rel &sop) {
+	const substrait::RelCommon* common = nullptr;
+	switch (sop.rel_type_case()) {
+	case substrait::Rel::RelTypeCase::kJoin:
+		common = &sop.join().common();
+		break;
+	case substrait::Rel::RelTypeCase::kProject:
+		common = &sop.project().common();
+		break;
+	default:
+		throw InternalException("Unsupported relation type " + to_string(sop.rel_type_case()));
```

Comment:
Almost every relation should be in this list.  I'd consider calling anything missing not yet implemented.

Reply:
Ah..! Thanks for pointing out, I missed this.

This also reminds me that I could avoid a project by using output mapping, whenever I only have to change the column order of a give relation.

Reply:
fixed it